packages/app/src/elements/import-dialog.ts

```diff
@@ -119,6 +137,7 @@ Github,"work,coding",https://github.com,john.doe@gmail.com,129lskdf93`)
                 this._items = await imp.asLastPass(file);
                 break;
             case imp.CSV.value:
+                this._itemColumns = await imp.asCSVColumns(file);
```

Comment:
Alternatively, I can make `.asCSV` return both `items` and `itemColumns`, so the file's only read once. I'll probably do that later.